src/Simulation/EntryPointDriver.Tests/Tests.fs

```diff
@@ -486,6 +486,7 @@ let private resourceSummary =
      T               0   0
      Depth           0   0
      Width           1   1
+     ExtraWidth      1   1
```

Comment:
It's not clear to me what ExtraWidth is, and what the value is of tracking it?

Reply:
I don't like this name either. This is what former "width" was called internally (see how it was renamed in ResourceEstimator.cs). We can come with a better name if necessary.
There's a value of tracking it (even if compatibility isn't an issue). Current width isn't a very useful metric for a single operation within execution, the old one is.

Reply:
@DmitryVasilevsky It would be good to understand why the width that is consistent with the depth is not a useful metric for a single operation? In particular, I think if you configure to optimize for width, I think you should recover the old width value (ExtraWidth), aren't you? I.e. if optimize for width is configured, aren't ExtraWidth and Width always the same? If not, where not and why?

Reply:
I suggest QubitCount instead of ExtraWidth.

Reply:
Sounds good. As per our discussion, I also assume that QubitCount will only show up in the CSV, but for the summary e.g. given by the ResourcesEstimator, only Depth and Width would be listed (applicable to the entire program).

Reply:
I've prepared a short summary describing Width, Depth and QubitCount.
I need to check the code for where QubitCount will show up.

Reply:
Thanks! Just one last comment in that regard, see here: https://github.com/microsoft/qsharp-runtime/pull/404#discussion_r509773163

Reply:
Addressed.